web/src/components/core/DevelopmentInfo.jsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { _ } from "~/i18n";
+
+/**
+ * Displays additional info when running in a development server
+ * @component
+ */
+export default function DevelopmentInfo () {
```

Comment:
Probably this deserves a better name (apart from a more complete test). And a better place, maybe it's time to create a _components/development/_ namespace to leave there components to be used only in development mode.

I'm thinking in an area in the sidebar for putting things related to Agama but only relevant when running in development mode.